release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -319,6 +319,15 @@ For the OVN-Kubernetes network plug-in, egress firewalls support audit logging u
 [id="ocp-4-12-nodes"]
 === Nodes
 
+[id="ocp-4-12-updating-interface-specific-list"]
+==== Updating the interface-specific safe list (Technology Preview)
+
+{product-title} now supports updating the default interface-specific safe `sysctls`.
+
+You can add or remove `sysctls` from the predefined list. When you add `sysctls` these are then available to be set appropriately across all nodes. Updating the interface-specific safe `sysctls` list is a Technology Preview feature only.
```

Comment:
` When you add `sysctls` these are then available to be set appropriately across all nodes. 
`I had to think for a second to understand this one. Maybe:

`Updated `sysctls` are available to set on all nodes.`

Reply:
I deferred to earlier peer review comment thanks @mmirecki